pkgs/development/python-modules/mailman-rss/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, dateutil, future, requests, beautifulsoup4 }:
+
+buildPythonPackage rec {
+  pname = "mailman-rss";
+  version = "0.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1brrik70jyagxa9l0cfmlxvqpilwj1q655bphxnvjxyganxf4c00";
+  };
+
+  propagatedBuildInputs = [ dateutil future requests beautifulsoup4 ];
+
+  checkPhase = ''
+    python -Wd -m unittest discover
```

I didn't see a test in the Pypi archive.
So they should be disabled `doCheck = false`.

Typically if there was an extensive test suite we'd do special measures
to get an archive that had them.

Btw, we'd always use the `interpreter` attribute when calling the python executable
```suggestion
    ${python.interpreter} -Wd -m unittest discover
```